aten/src/ATen/SparseCsrTensorImpl.cpp

```diff
@@ -113,4 +119,21 @@ void SparseCsrTensorImpl::set_member_tensors(
   sizes_and_strides_.set_sizes(size);
   refresh_numel();
 }
+
+IntArrayRef SparseCsrTensorImpl::strides() const {
```

Comment:
Now that it's quick to merge a PR we can pull this out into its own small piece and land it right away if you want.

Reply:
I moved into a separate PR: https://github.com/pytorch/pytorch/pull/75499. It should be merged first, then this one.